game/src/info/building.rs

```diff
@@ -9,8 +9,16 @@ use widgetry::{Color, EventCtx, Line, Text, TextExt, Widget};
 use crate::app::App;
 use crate::info::{header_btns, make_table, make_tabs, Details, Tab};
 
-pub fn info(ctx: &mut EventCtx, app: &App, details: &mut Details, id: BuildingID) -> Vec<Widget> {
-    let mut rows = header(ctx, app, details, id, Tab::BldgInfo(id));
+pub fn info(ctx: &mut EventCtx, app: &App, details: &mut Details, id: BuildingID) -> Widget {
+    Widget::custom_col(vec![
```

Comment:
Since the existing info panel content building was pretty idiomatic, the changes here are pretty representative of the entire PR.

Instead of returning a Vec<Widget>, we return a single widget, composed of two items:
1. the header (which includes the tab buttons) 
2. the tab body content